test/unit/vl-map-search.test.html

```diff
@@ -205,6 +322,207 @@
         assert.equal(element.getTranslation('select.no_more_options'), text4);
       });
     });
+
+    suite('Lambert-coördinaat', () => {
+      test('of - een vrije tekst, geeft "undefined"', () => {
+        expect(LambertCoordinaat.of('test')).to.be.undefined;
+      });
+
+      test('of - "null", geeft "undefined"', () => {
+        expect(LambertCoordinaat.of(null)).to.be.undefined;
+      });
+
+      test('of - "undefined", geeft "undefined"', () => {
+        expect(LambertCoordinaat.of(undefined)).to.be.undefined;
+      });
+
+      test('of - een lege string, geeft "undefined"', () => {
+        expect(LambertCoordinaat.of('')).to.be.undefined;
+      });
+
+      test('of - een geldige coördinaat, geeft een LambertCoordinaat', () => {
+        const lambertCoordinaat = LambertCoordinaat.of('123456.78, 345232.04');
+
+        expect(lambertCoordinaat).not.to.be.undefined;
+        expect(lambertCoordinaat).to.deep.equal({
+          _x: 123456.78,
+          _y: 345232.04,
+        });
+        expect(lambertCoordinaat.toString()).to.be.equal('123456.78, 345232.04');
```

Comment:
Ik zou voor de toString een afzonderlijke test voorzien. Wanneer er nu een probleem in de toString ontstaat, falen er te veel testen die niet exact aangeven wat er fout loopt.